src/pages/sponsor.module.css

```diff
@@ -0,0 +1,77 @@
+.container {
+  margin: 0 auto;
+  padding: 24px 24px;
+  max-width: 960px;
+  background-color: #121212;
+  color: white;
+  align-items: center;
+}
+
+.dispImage {
+  object-fit: contain;
+  object-position: center;
+  width: 100%;
+  max-width: 500px;
+  height: auto;
+}
+
+.h2alt {
+  margin-top: 20px;
+}
+
+.imageContainerLeft {
+  margin-top: 0;
+  margin-left: 30px;
+  display: flex;
+  flex: 0 0 33%;
+}
+
+.imageContainerRight {
+  margin-top: 20px;
+  margin-right: 30px;
+  display: flex;
+  flex: 0 0 37%;
+}
+
+.rowText {
+  display: flex;
+  flex-direction: row;
+}
+
+.section {
+  background-color: #262626;
+  color: white;
+}
+
+@media only screen and (max-width: 768px) {
+  .h2alt, h2 {
+    text-align: center;
+    margin-bottom: 5px;
+  }
+
+  .dispImage {
+    display: block;
+    width: 75%;
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  .imageContainerRight {
+    margin: 0;
+    width: 100%;
+  }
+  
+  .imageContainerLeft {
```

Comment:
Same feedback as above - and this instance is a good example of a reason why! If a new dev takes over the code base, and they add more content to this page, they might apply this CSS class to another image to lay it out to the ~left~ right. They'd then test it on mobile and be confused as to why the image disappeared, when they thought all the class would do is lay it out to one side.. The design decision is specific to this image (I assume, since the other one doesn't disappear), so it should be named to make that clear.

(Also, these names are backwards - .`imageContainerLeft` is laying the image to the right)